src/auth.js

```diff
@@ -0,0 +1,24 @@
+const { OAuth2Client } = require('google-auth-library');
+
+const CLIENT_ID =
+  '270040816063-djog7f8mpvt4m162ak4n04bjmftg1bhc.apps.googleusercontent.com';
```

Comment:
This is publicly available, so I don't think we need to be paranoid about hardcoding this. But it might be good later on to make it an environment variable